.github/workflows/announce-release.yml

```
@@ -0,0 +1,33 @@
+  push:
+    branches:
+      - main
+    path:
+      - 'CHANGELOG.md'
+
+permissions:
+  pull-requests: read
+
+jobs:
+  post_to_mastodon:
+    if: contains(github.event.pull_request.title, 'chore(main): release')
```

actionlint (1.7.4)

could not parse as YAML: yaml: line 15: mapping values are not allowed in this context

(syntax-check)